.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug report
+description: Create a report to help us improve
+title: "[BUG]: "
+labels: ["bug"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this bug report!
+
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version are you using?
+      options:
+        - v6 (Beta)
+        - v5 (Stable)
+    validations:
+      required: true
+
+  - type: dropdown
+    id: platforms
+    attributes:
+      label: What platforms are you having the problem on?
+      multiple: true
+      options:
+        - iOS
+        - Android
+        - Windows
+        - visionOS
+        - Android TV
+        - Apple tvOS
+  
+  - type: dropdown
```

Comment:
maybe comment this until new architecture is supported ?

Reply:
I think we should leave interop layer option